assets/training/model_management/src/run_model_preprocess.py

```diff
@@ -140,6 +141,14 @@ def run():
                 AzureMLError.create(UnsupportedTaskType, task_type=args.task_name,
                                     supported_tasks=list(supported_tasks))
             )
+    if hf_model_args is None:
```

Comment:
Please first check if model folder contains .py files or not.

Reply:
updated the code changes